portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includePageNav.vm

```diff
@@ -19,8 +19,8 @@
 	<div id="toolSubsitesContainer">
 		<nav id="toolMenu" role="navigation" aria-label="${rloader.sit_toolshead}" class="Mrphs-toolsNav__menu">
 			<ul>
-				<li class="Mrphs-toolsNav__menuitem Mrphs-collapseTools #if ( ${toolsCollapsed} ) min #else max #end" style='display: none;'>
-					<button type="button" tabindex="0" aria-pressed="#if ( ${toolsCollapsed} )false#elsetrue#end" title="${rloader.sit_toggle_nav}" class="Mrphs-toggleNav js-toggle-nav btn-link  #if ( ${toolsCollapsed} ) min #else max #end">
+				<li id="toolsNav-toggle-li" class="Mrphs-toolsNav__menuitem Mrphs-collapseTools #if ( ${toolsCollapsed} ) min #else max #end" style='display: none;'>
+					<button type="button" tabindex="0" aria-pressed="#if ( ${toolsCollapsed} ) false #else true #end" title="${rloader.sit_toggle_nav}" class="Mrphs-toggleNav js-toggle-nav btn-link  #if ( ${toolsCollapsed} ) min #else max #end">
```

Comment:
The extra spaces here were a bad change ..... This violates accessibility compliance looking for "true" and not " true "